src/index.js

```diff
@@ -13,15 +14,52 @@ const traverse = require('traverse');
 
 const replacement = '--REDACTED--';
 
+/**
+ * Apply serializers.
+ */
+
+function applySerializers(values, serializers) {
+  for (const key in serializers) {
+    if (values[key] === undefined) {
+      return;
+    }
+
+    try {
+      values[key] = serializers[key](values[key]);
+    } catch (error) {
+      values[key] = `Anonymize ERROR: Error while applying ${key} serializer: ${error.message}`;
+    }
+  }
+}
+
 /**
  * Module exports.
  */
 
-module.exports = (whitelist = []) => {
+module.exports = (whitelist = [], options = {}) => {
+  // Add error serializers as default serializers.
```

Comment:
Why? If I'm understanding this correctly we want this to be executed always as it may be related to node particularities such as the `error.stack` being a non enumerable property.